tests/Unit/Domain.LinnApps.Tests/PurchaseOrderDeliveryServiceTests/WhenUpdatingDetailsRemovingFromDeliveryTotal.cs

```diff
@@ -0,0 +1,138 @@
+﻿namespace Linn.Purchasing.Domain.LinnApps.Tests.PurchaseOrderDeliveryServiceTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    using FluentAssertions;
+    using FluentAssertions.Extensions;
+
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders;
+    using Linn.Purchasing.Domain.LinnApps.PurchaseOrders.MiniOrders;
+
+    using NSubstitute;
+
+    using NUnit.Framework;
+
+    public class WhenUpdatingDetailsRemovingFromDeliveryTotal : ContextBase
+    {
+        private PurchaseOrder order;
+
+        private PurchaseOrderDetail line;
+
+        private IEnumerable<PurchaseOrderDelivery> updatedDeliveries;
+
+        private IEnumerable<PurchaseOrderDelivery> result;
+
+        private decimal vatAmount;
+
+        private decimal baseVatAmount;
+
+        [SetUp]
+        public void SetUp()
+        {
+            this.vatAmount = 543;
+            this.baseVatAmount = 544;
+            //add new deliveries 
+            this.updatedDeliveries = new List<PurchaseOrderDelivery>
+                                         {
+                                             new PurchaseOrderDelivery
+                                                 {
+                                                     DeliverySeq = 1,
+                                                     OurDeliveryQty = 10,
+                                                     BaseOurUnitPrice = 11,
+                                                     DateRequested = 25.March(2022),
+                                                     AvailableAtSupplier = "N"
+                                                 }
+                                         };
+
+
+            // Order will be updated to delivery total. Simulate removing deliveries.  
+            this.line = new PurchaseOrderDetail
+            {
+                Line = 1,
+                OrderUnitPriceCurrency = 666,
+                OrderConversionFactor = 1.5m,
+                BaseOurUnitPrice = 111,
+                PurchaseDeliveries = new List<PurchaseOrderDelivery>(),
```

Comment:
If this test is for the scenario where you are removing deliveries, then shouldn't this list have more deliveries in it than the 'updatedDeliveries' list?